src/remote_prune

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import subprocess
+import json
+import yaml
+import collections
+import datetime
+import os
+from dateutil.relativedelta import relativedelta
+
+from cosalib.aliyun import remove_aliyun_image
+from cosalib.gcp import remove_gcp_image
+from cosalib.azure import remove_azure_image
+from cosalib.prune import fetch_build_meta, delete_build
+
+from cosalib.aws import deregister_ami, delete_snapshot
+
+Build = collections.namedtuple("Build", ["id", "timestamp", "images", "arches"])
```

Comment:
note that we do have a Build() class in cosalib as well. Not sure if that would be useful here to use that instead or not.

Reply:
We do indeed have that but they are changed according to those files. It seemed better to create a new variable since we are using `cloud_images` here.